src/databricks/labs/remorph/reconcile/connectors/data_source.py

```diff
@@ -20,11 +20,18 @@ def __init__(self, source: str, spark: SparkSession, ws: WorkspaceClient, scope:
         self.scope = scope
 
     @abstractmethod
-    def read_data(self, schema_name: str, catalog_name: str, query: str, table_conf: Tables) -> DataFrame:
+    def read_data(
+        self, catalog_name: str, schema_name: str, query: str, jdbc_reader_options: JdbcReaderOptions
```

Comment:
```suggestion
        self, catalog: str, schema: str, query: str, options: ReaderOptions
```
As long as you are changing signature, make arg names shorter

Reply:
fixed.